MOM6-interface/mom6_files.cmake

```diff
@@ -24,12 +24,13 @@ list(APPEND mom6_src_files
   MOM6/src/core/MOM_CoriolisAdv.F90
   MOM6/src/core/MOM_PressureForce.F90
   MOM6/src/core/MOM_PressureForce_Montgomery.F90
-  MOM6/src/core/MOM_PressureForce_analytic_FV.F90
+  MOM6/src/core/MOM_PressureForce_FV.F90
```

Comment:
@jiandewang 
Please revert the permissions to 100644 on this file and all other text files

Reply:
chmod 644 done, thanks for pointing out this